docs/source/simple_up.rst

```diff
@@ -0,0 +1,102 @@
+Up
+--
+
+It's time to provision your first LinchPin resources.
+
+.. code-block:: bash
+
+    1   [/tmp/simple]$ linchpin up
```

Comment:
@herlo 
linchpin up block here has line numbers , 
but linchpin destroy does not . 
To maintain consistency could you please remove the line numbers here ! or add line numbers to linchpin destroy

Reply:
Actually, the purpose for the line numbers is to explain what things are in the components. I'm not sure I need to add line numbers to linchpin destroy since everything that is explained is covered in the `linchpin up` section. If you render the documentation, you'll see superscript numbers that correspond to the line numbers. This is what the `:sup:8` is for [here](https://github.com/CentOS-PaaS-SIG/linchpin/pull/758/files#diff-05d8e36df29d8bddd5336fe0290917c1R39).

Can you tell me why it would be needed for the other examples on this page?

Reply:
It might be needed if we would like to refer . Since for dummy provider destroy is self explanatory it not needed as you explained . Approving it now .